templates/src/project/sln/Directory.Build.Props

```diff
@@ -0,0 +1,14 @@
+﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup Condition="'$(Configuration)' == ''">
+        <Configuration>Debug</Configuration>
+    </PropertyGroup>
+        
+    <PropertyGroup>
+        <OpenTapVersion>9.19.0</OpenTapVersion>
```

Comment:
Should this be $(GitVersion)

Reply:
Possibly, i honestly dont remember if there was a reason to leave it like this. Maybe because we want the lowest possible tap version for templates to allow more backwards compatibility?